Makefile

```diff
@@ -57,7 +57,7 @@ compilejsi18n:
 compile-react-i18n:
 	@ echo "## Generating React translation catalogs ##"
 	@ mkdir -p build/locale
-	@ python manage.py compilejsi18n -d react
+	@ python manage.py compilejsi18n -d react -f json
```

Comment:
I'm not sure when it's the right time but this command just *has* to work perfectly before every deployment. 

I can help. There should probably be a Django system check that asserts that for every locale in `settings.ACCEPTED_LANGUAGES` we expect there to be a non-corrupt `{settings.BASE_DIR}/static/jsi18n/{locale}/react.json`. Because if we have that, we can be confident the `get_localization_data` function never fails. 

Is that something you want me to help with.

Reply:
It sounds like a good idea to add something like this, and I'd welcome your help since I don't know what a "system check" is. But I hope we don't need to block this PR on adding that check.

Reply:
Let's move that discussion to https://github.com/mozilla/kuma/pull/5425